browser/src/control/jsdialog/Widget.Combobox.js

```diff
@@ -237,11 +237,25 @@ JSDialog.combobox = function (parentContainer, data, builder) {
 
 		resetSelection();
 		for (var i in entries) {
-			if (entries[i] == this.value) {
+			if (entries[i].text == this.value) {
 				entries[i].selected = true;
 				break;
 			}
 		}
+		// check for drop down is in open state or not.
+		// If open then we should make focus in entries field for Arrow key navigation
```

Comment:
why do we need to add here some additional focusing?
maybe we can just `event.preventDefault()` to stop processing any other callback which will changer focus?

Reply:
I see, we can still be with focus on the dropdown button or combobox entry, but when using arrow key - we jump to selected entry in the list - which is expected